connect-button/src/main/java/com/ifttt/connect/ui/BaseConnectButton.java

```diff
@@ -648,13 +627,16 @@ public void onAnimationEnd(Animator animation) {
                 ((StartIconDrawable) iconImg.getBackground()).setBackgroundColor(worksWithService.brandColor);
 
                 if (buttonApiHelper.shouldPresentCreateAccount(getContext())) {
-                    Animator completeProgress =
-                            progressView.progress(0.5f, 1f, getResources().getString(R.string.ifttt_creating_account),
-                                    ANIM_DURATION_LONG);
```

Comment:
can remove unused string `ifttt_creating_account`

Reply:
Good catch, this is not intentional, we should still have the state.

Reply:
https://github.com/IFTTT/ConnectSDK-Android/pull/137/commits/926d3bec98f8d22edb2adc0aa13001a5c1e6c653